src/data/javascript-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const javascriptQuiz = [
+  {
+    Question: "The Math.random() method returns a random number in what range?",
+    Answer: "Between 0 and 1 (including 0; not including 1).",
+    Distractor1: "Between 0 and 1 (including neither 0 or 1).",
+    Distractor2: "Between 0 and 1 (including 1; not including 0).",
+    Distractor3: "Between 0 and 100 (including 0; not including 100).",
```

Comment:
I think this wording will be a little bit cleaner

```suggestion
    Answer: "Between 0(inclusive) and 1(exclusive).",
    Distractor1: "Between 0 and 4",
    Distractor2: "Between 1 and 2",
    Distractor3: "Between 0 and 100",
```